mkdocs.yml

```diff
@@ -32,5 +32,164 @@ extra_javascript:
+      - Timesheets (Unanet): 050-how-we-work/tools/unanet.md
+      - Security Policy: 030-policies/security.md
+      - Employment policies (US employees):
+          - Employment: 040-employee-handbook-us/employment.md
+          - Introductory period: 040-employee-handbook-us/introductory-period.md
+          - Anti-harassment: 040-employee-handbook-us/anti-harassment-policies.md
+          - Covid 19 safety: 040-employee-handbook-us/covid19safety.md
+      - Code of conduct: 030-policies/code-of-conduct.md
+      - Workplace guidelines: 030-policies/workplace-guidelines.md
+      - Annual performance review: 030-policies/annual-review-process.md
+      - Performance management: 030-policies/performance-management.md
+      - Leaving Civicactions: 030-policies/leaving-civicactions.md
+      - Health, Safety and Security Policy: 030-policies/health-safety-security.md
+      - New hire orientation:
+          - Welcome to CivicActions!: 010-welcome-to-civicactions/welcome.md
+          - Buddy program: 010-welcome-to-civicactions/training/buddy-program.md
+          - Elevator pitch: 020-about-us/elevator-pitch.md
+          - Free and open source software (FOSS): 010-welcome-to-civicactions/training/intro-open-source.md
+          - Meetings and meeting tools: 010-welcome-to-civicactions/training/meetings-and-meeting-tools.md
+          - People planning: 010-welcome-to-civicactions/training/people-planning.md
+          - Security for everyone: 010-welcome-to-civicactions/training/security-training.md
+          - Training resources: 050-how-we-work/training-resources.md
+          - Video call best practices: 050-how-we-work/video-call-best-practices.md
+          - Virtual workplace basics: 050-how-we-work/virtual-workplace-basics.md
+  - Common practices and tools:
+      - Agile:
+          - Agile overview: 050-how-we-work/agile-overview.md
+          - Agile practice: 050-how-we-work/agile-practices/README.md
+          - Backlog refinement: 050-how-we-work/agile-practices/backlog-refinement.md
+          - Daily scrum calls: 050-how-we-work/agile-practices/daily-scrum-calls.md
+          - How to create tickets: 050-how-we-work/tools/tickets-cards.md
+          - Sprint cycle: 050-how-we-work/agile-practices/sprint-cycle.md
+          - Sprint demo: 050-how-we-work/agile-practices/sprint-demo.md
+          - Sprint planning meetings: 050-how-we-work/agile-practices/sprint-planning-meetings.md
+          - Sprint retrospectives: 050-how-we-work/agile-practices/sprint-retrospectives.md
+      - Balance scores: 050-how-we-work/balance-scores.md
+      - Gold Star Program: 050-how-we-work/gold-star-program.md
+      - Security awareness and tools:
+          - Awareness and tools: 100-security/awareness.md
+            - Setting up and using Yubikey: 100-security/yubikey/README.md
+              - Yubikey and Linux: 100-security/yubikey/linux.md
+              - Yubikey and Mac OS: 100-security/yubikey/macosx.md
```

Comment:
@ContentBrewmaster I believe the is causing the error in the checks. I think the identation should look like this:

```
          - Awareness and tools: 100-security/awareness.md
          - Setting up and using Yubikey:
              - 100-security/yubikey/README.md
              - Yubikey and Linux: 100-security/yubikey/linux.md
              - Yubikey and Mac OS: 100-security/yubikey/macosx.md
```

Reply:
@ContentBrewmaster thanks for fixing these. I made one minor change and I think we are good with those.